sql/004.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE `perks` (
+  `id` int NOT NULL AUTO_INCREMENT,
+  `name` varchar(60) NOT NULL,
+  `point_price` int NOT NULL,
+  `starts` date NOT NULL,
+  `ends` date NOT NULL,
+  `combination with other offers` tinyint NOT NULL,
+  `perk photo` varchar(100) NOT NULL,
```

Comment:
Have you got a chance to sync with @anavianm and @ankoanju ? Are they going to store a URL here? If yes, then 100 characters may not be enough. And you will need a totally different field if they need to store binary data.